code/Controllers/SilverStripeNavigatorItem_Unversioned.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\CMS\Controllers;
+
+use Config;
+use SilverStripe\CMS\Controllers\SilverStripeNavigatorItem;
+use SilverStripe\Core\Convert;
+use SilverStripe\Versioned\Versioned;
+
+class SilverStripeNavigatorItem_Unversioned extends SilverStripeNavigatorItem
+{
+    public function getHTML()
```

Comment:
For all of these methods, would you be able to add php dockblocks for all the `@param` + `@return`, and php7 return types, though omit php7 return types if existing parent class(es) API don't define php7 return types

Reply:
Can do, though they're just copied from the superclass. I had omitted them originally because the PHPDoc will typically inherit from the superclass if omitted. Most methods in this class are overriding the methods in `SilverStripeNavigatorItem` and with the exception of `canView` I don't think there's anything to add to the PHPDoc from that class.

Let me know if you want me to add PHPDocs for inherited methods anyway and I'll do so.
In the meantime I'll add any for non-inherited methods and where I feel I have something to add.